src/applications/mhv/medical-records/components/RecordList/AllergyListItem.jsx

```diff
@@ -13,15 +13,15 @@ const AllergyListItem = props => {
           className="record-list-item vads-u-border-color--gray-light vads-u-border--0 vads-u-background-color--gray-lightest card"
           data-testid="record-list-item"
         >
-          <Link
-            to={`/allergies/${record.id}`}
-            className="vads-u-margin--0 no-print"
-            aria-label={`View details for ${record.name} on ${record.date}`}
-          >
-            <h3 className="vads-u-font-size--h4 vads-u-line-height--4">
+          <h3 className="vads-u-font-size--h4 vads-u-margin--0 vads-u-line-height--4">
+            <Link
+              to={`/allergies/${record.id}`}
+              className="vads-u-margin--0 no-print"
+              aria-label={`${record.name} on ${record.date}`}
```

Comment:
I noticed you removed "View details for" from the aria-label, although it wasn't in the ticket. Did that request come from UCD?

Reply:
@mmoyer-va Apologies, I neglected to include it in the commits. Yes, Bobby instructed me to remove it.